main.tf

```diff
@@ -27,6 +26,20 @@ variable "network_ranges" {
   description = "CIDR ranges to use for the cluster deployment."
 }
 
+variable "gke_initial_node_count" {
+  type    = number
+  default = 1
```

Comment:
Add a description so that terraform-docs adds it into the docs for you

Reply:
nice - I didn't realize that

thanks!